src/components/OrderableListV2/OrderableListV2.spec.tsx

```diff
@@ -0,0 +1,217 @@
+        const stubbedOnMove = cy.stub().as("onMove");
+        const dataTransfer = new DataTransfer();
+
+        mount(<OrderableListWithDefaultProps onMove={stubbedOnMove} />);
+
+        cy.get(DRAGGABLE_ITEM).first().trigger("dragstart", { dataTransfer }).trigger("drag");
+        cy.get(LIST_ID)
+            .trigger("dragenter", { ...getCoordinatesByIndex(4, "Before"), dataTransfer })
+            .trigger("dragover", { ...getCoordinatesByIndex(4, "Before"), dataTransfer });
+        cy.get(INSERTION_INDICATOR).should("exist").and("have.length", 1);
+        cy.get(LIST_ID).trigger("drop", { dataTransfer, force: true });
+        cy.get("@onMove").should("have.been.called");
+    });
+
+    it("Hides insertion indicator if position change is original index", () => {
+        mount(<OrderableListWithDefaultProps />);
+
+        cy.get(DRAGGABLE_ITEM).each(($el, index) => {
+            const dataTransfer = new DataTransfer();
+            cy.wrap($el).trigger("dragstart", { dataTransfer }).trigger("drag");
+            cy.get(LIST_ID)
+                .trigger("dragenter", { ...getCoordinatesByIndex(index, "Before"), dataTransfer })
+                .trigger("dragover", { ...getCoordinatesByIndex(index, "Before"), dataTransfer });
+            cy.get(INSERTION_INDICATOR).should("have.length", 0);
+            cy.get(LIST_ID)
+                .trigger("dragenter", { ...getCoordinatesByIndex(index, "After"), dataTransfer })
+                .trigger("dragover", { ...getCoordinatesByIndex(index, "After"), dataTransfer });
+            cy.get(INSERTION_INDICATOR).should("have.length", 0);
+        });
+    });
+
+    it("Passes on correct drag state to each item", () => {
+        const renderWithDragState = (_item: GridNode<TestItem>, { componentDragState }: DragProperties) => (
+            <div style={{ height: `${ITEM_HEIGHT}px` }}>{componentDragState}</div>
+        );
+
+        mount(<OrderableListWithDefaultProps renderContent={renderWithDragState} />);
+
+        cy.get(DRAGGABLE_ITEM).each(($el, index, $list) => {
+            const dataTransfer = new DataTransfer();
+            const dropIndex = index + 2 > $list.length ? index - 2 : index + 1;
+            cy.wrap($el).should("have.text", ItemDragState.Idle).trigger("dragstart", { dataTransfer }).trigger("drag");
+            cy.wrap($el).should("have.text", ItemDragState.Dragging);
+            cy.get(LIST_ID)
+                .trigger("dragenter", { ...getCoordinatesByIndex(dropIndex, "After"), dataTransfer })
+                .trigger("dragover", { ...getCoordinatesByIndex(dropIndex, "After"), dataTransfer });
+            cy.get(INSERTION_INDICATOR).should("exist").and("have.length", 1);
+            cy.get(LIST_ID).trigger("drop", { dataTransfer, force: true });
+            cy.wrap($el).trigger("dragend", { dataTransfer, force: true });
+            cy.wrap($el).should("have.text", ItemDragState.Idle);
+        });
+    });
+
+    it("Has keyboard accessibility", () => {
```

Comment:
Does it make sense to split this into multiple tests?

Reply:
Maybe. But this will be deleted for now, as we don't have keyboard control in the current implementation. There will be a separate ticket for that.